plugin/ui/completion.py

```diff
@@ -84,24 +132,46 @@ def show_next_completion(self) -> None:
         """Show the next completion."""
         self.show(completion_index=self.completion_index + 1)
 
+    def handle_selection_change(self) -> None:
+        if not self.is_phantom() or not self.is_visible:
+            return
+
+        self.hide()
+
+    def handle_text_change(self) -> None:
```

Comment:
I know that this is is exacrtly the same as the one above, but I am planning to extend it in a follow-up and handle text changes when the phantom completion is visible